R/convenience-functions.R

```diff
@@ -0,0 +1,48 @@
+#' @title Add transformations
+#'
+#' @description Add transformations of the forecasts and observations for
```

Comment:
I think you can give a little more detail here as this is extremely sparse and doesn't really make people want to check the reference.